posts/2022-06-03-liberty-deep-dive.adoc

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Introducing a hands-on technical deep dive on Liberty!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Introducing a hands-on technical deep dive on Liberty! - OpenLiberty.io
+seo-description: Introducing an end-to-end tutorial on cloud-native Java application development and deployment using Jakarta EE, MicroProfile and Liberty. The tutorial covers building RESTful microservice, documenting your REST API, externalizing configuration, persisting data to a database, securing application using JSON Web Tokens (JWT), adding health checks, providing metrics, testing using Testcontainers, building container images, and deploying to Kubernetes using Operator.
+blog_description: "Introducing an end-to-end tutorial on cloud-native Java application development and deployment using Jakarta EE, MicroProfile and Liberty"
+open-graph-image: https://openliberty.io/img/blog/deepdive.png
+---
+= Introducing a hands-on technical deep dive on Liberty
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+We are excited to announce the release of our newest tutorial, A Technical Deep Dive on Liberty.
+
+A Technical Deep Dive on Liberty is an end-to-end tutorial that walks you through how to develop and deploy cloud-native Java applications in Jakarta EE and MicroProfile. The tutorial is aimed at Java developers, from Liberty novices to those more experienced with writing applications to run on Liberty.
+
+In A Technical Deep Dive on Liberty, you will learn how to:
+
+* build a RESTful microservice on Liberty with Jakarta EE and MicroProfile
+* easily document your REST API for developers to consume your microservice
+* externalize your configuration to make your application portable and easier to update
+* persist data to a database
+* secure your application using JSON Web Tokens (JWT) so that only authenticated and authorized applications can request information from your microservice
+* add health checks to easily report the statuses of your microservice (for orchestration decision)
+* provide metrics to monitor activity and performance of your application
+* build a container image for your Java microservice
+* deploy the container image to Kubernetes using the Open Liberty Operator
+* test your applications easily with containers (for true-to-production testing)
```

Comment:
feels like all of these should start with a capital letter

Reply:
I missed this when the list was updated- it originally had a period at the end. Fixing